docs/docs/overview/quickstart/get-started-with-node.js.md

```diff
@@ -0,0 +1,244 @@
+
+![send_test_email.png](https://res.cloudinary.com/dxc6bnman/image/upload/v1686776583/send_test_email_ngzmth.png)
+
+You should get an email within seconds. Yaaay, you have successfully sent your first notification via the Novu dashboard! Now, let’s take it a step further to trigger notifications via code.
+
+## Create A Subscriber
+
+The recipients of a triggered notification are called subscribers.
+
+Click “Subscribers” on the left sidebar of the Novu dashboard to see all subscribers. By default, the dashboard will display a subscriber, as you were added automatically during sign-up.
+
+![subscriber_id.png](https://res.cloudinary.com/dxc6bnman/image/upload/v1686776584/subscriber_id_tzkwjh.png)
+
+Now, let's create a subscriber on Novu. Copy and paste the following code to do so:
+
+```jsx
+// Create a subscriber
+const subscriberId = '7789'; // Replace this with a unique user ID that matches your database.
+await novu.subscribers.identify(subscriberId, {
+  email: 'abc@gmail.com', // optional
+  firstName: 'John', // optional
+  lastName: 'Doe', // optional
+  phone: '', // optional
+  avatar: '', // optional
+  locale: '', // optional
+  data: { customKey1: 'customVal1', customKey2: 'customVal2' }, // optional
+});
+```
+
+Run the code in your terminal like so:
+
+```bash
+node index.js
+```
+
+You should see the subscriber on your Novu dashboard.
+
+![Screenshot 2023-05-14 at 11.06.38.png](https://res.cloudinary.com/dxc6bnman/image/upload/v1685466979/guides/Screenshot_2023-05-14_at_11.06.38_ugvmc0.png)
+
+I’d like to publicly announce that `abc@gmail.com` is a random unlikely email your users will have. To update this to an alternative email, you can call the `updateSubscriber` method like so:
+
+```jsx
+// Update subscriber detail
+await novu.subscribers.update('7789', {
+  // new email
+  email: 'validemail@gmail.com', // optional
+  // new phone
+  phone: '+19874567832', // optional
+});
+```
+
+Other valid fields that can be updated are `phone`, `avatar`, and `data` . The `data` field can accept an array of metadata that you want to attach to the subscriber.
+
+<aside>
+🔑  **Note:** To make all of your app users subscribers, you need to programmatically add them to Novu.
+
+</aside>
+
+## Trigger A Notification
+
+Copy and paste the following code into your app to trigger a notification:
+
+```jsx
+const notificationTemplateId = 'first-email';
+
+novu.trigger(notificationTemplateId, {
+  to: {
+    subscriberId: '7789',
+  },
+  payload: {},
+});
+```
+
+Before running the code, make sure you understand the following:
+
+- The value of `notificationTemplateId` should be the notification template’s trigger ID/slug.
+
+![trigger_id.png](https://res.cloudinary.com/dxc6bnman/image/upload/v1686776585/trigger_id_xkhsx7.png)
+
+- The value of `payload` is an array of the data that you want to be dynamically injected into the notification template content.
+- The value of `subscriberId` is the id of the subscriber on Novu. Replace `7789`
+
```

Comment:
Please remove the unneeded space.  Replace `7789` with your subscriber ID.

Reply:
Done